scripts/webidl/convert.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+const fs = require("fs");
+const path = require("path");
+
+const recursive = require("recursive-readdir");
+const webidl2js = require("webidl2js");
+
+const optimist = require("optimist")
+  .usage("Pull all webidl segments froma a web page")
+  .string("input")
+  .alias("input", "i")
+  .describe("input", "the input file or directory")
+  .string("implDir")
```

Comment:
Command line flags are traditionally kebab-case, not camelCase. So --impl-dir etc.